Makefile

```diff
@@ -0,0 +1,53 @@
+ENV_DIR=.env
+
+ifeq ($(OS),Windows_NT)
+	IN_ENV=. $(ENV_DIR)/Scripts/activate &&
+else
+	IN_ENV=. $(ENV_DIR)/bin/activate &&
+endif
+
+all: test lint docs artifacts
+
+env: $(ENV_DIR)
+
+test: build plain-test
+
+plain-test:
+	$(IN_ENV) tox -p all
+
+artifacts: build_reqs sdist
+
+$(ENV_DIR):
+	virtualenv $(ENV_DIR)
+
+build_reqs: env
+	$(IN_ENV) pip install sphinx tox coverage
```

Comment:
Do we need coverage in the outer environment? Seems like something for the tox environment(s)

Reply:
True that. Changed